config/initializers/secret_token.rb

```diff
@@ -4,4 +4,4 @@
 # If you change this key, all old signed cookies will become invalid!
 # Make sure the secret is at least 30 characters and all random,
 # no regular words or you'll be exposed to dictionary attacks.
-CampusCodefest::Application.config.secret_token = '575ca50571c5ba792cac840e79f7b8b8362d420f7c56cdd8ae3040c994d9e459431e3f33f0138bc68e3b0e54ee1de9f9d4448e6b065208fed2ae2eb7f28ade4a'
+CampusCodefest::Application.config.secret_token = ENV['CCF_SECRET_TOKEN']
```

Comment:
Agree with all points, it feels like config file makes sense here for consistency, but the CI point is valid.